BeehiveCommunity/Frontend_Mentor
# Fix Navbar Line Connection Bug and Update Design

## What does this PR do?

This PR enhances the website's navbar by correcting links, adding a brand name, improving spacing and alignment, and updating the hover effect to turn items black for better visibility, resulting in a more polished and user-friendly design.

Fixes #282 

![image](https://github.com/user-attachments/assets/fe925d9d-9424-4773-8e72-1824dce84e70)

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements) 
- Breaking change (fix or feature that would cause existing functionality to not work as expected) 

## How should this be tested?

To test this PR, verify that the navbar displays correctly, check that all links navigate to the appropriate pages, and ensure that hovering over navbar items changes their color to black. Additionally, confirm that the spacing between items is consistent and visually appealing.

## Mandatory Tasks

- Self-reviewed the code to ensure quality and functionality. All changes are consistent and align with project standards.
